driver.sh

```diff
@@ -34,12 +33,28 @@ build_linux() {
   cd linux
   export ARCH=arm64
   export CROSS_COMPILE=aarch64-linux-gnu-
+  # TODO: do we really want to be building the kernel from scratch every time?
+  # Rerunning ninja above will not rebuild unless source files change.
   make CC=$clang mrproper
   make CC=$clang defconfig
   make CC=$clang -j`nproc`
   cd -
 }
 
+build_root() {
+  mkdir -p buildroot/overlays/etc/init.d/
+  cp -f inittab buildroot/overlays/etc/.
+  cp -f S50yolo buildroot/overlays/etc/init.d/.
+
+  cd buildroot
+  make defconfig BR2_DEFCONFIG=../buildroot.config
+  make
```

Comment:
probably want to `make -j$(nproc)` (GitHub hates the ticks in markdown lol) this too just for consistency.

A make wrapper might not be a bad idea either

```
function make() {
  command make -j`nproc` "$@"
}
```

Reply:
There's no rendering issue with github's markdown renderer.  I guess I'm curious if folks using bash have an issue with the backticks?  No, seems like backticks work there.

Does buildroot support `-j`?

Reply:
Ah ```make -j`nproc` ``` works

Shellcheck recommends using `$(...)` over backticks and that's what I always use personally: https://github.com/koalaman/shellcheck/wiki/SC2006

I thought it did but maybe not.

Reply:
done